conf/machine/rpi4-test.extra.conf

```diff
@@ -0,0 +1,21 @@
+require conf/machine/include/rpi.inc
+
+# FIXME: just for testing purposes, remove before merging!
+DISTRO_FEATURES:append = " 3g tpm2"
+MACHINE_FEATURES:append = " 3g tpm2"
+
+# adu compatibility id
+ADU_DEVICEPROPERTIES_COMPATIBILITY_ID = "2"
+
+# wic: how to partition
+WKS_FILE = "omnect-os.rpi4.wks.in"
+
+# u-boot boot cmd config vars
+UBOOT_ARCH = "arm64"
+
+OMNECT_PART_SIZE_ROOTFS    ?= "900000"
+
+OMNECT_ONLINE_INTERFACES = "${OMNECT_ETH0}"
```

Comment:
why not WWAN0 resp. WLAN0?

Reply:
yes, in the end somewhere has to be defined which interfaces are available for each interface class, and for Ethernet this can vary between one and three interfaces now.
this depends on the target only, at least at the moment, it might change if we supported several models of the same hardware with varying number of Ethernet interfaces.

of course I could change that to always assume OMNECT_ETH0 being set, so that in omnect-os-distro.conf it can be unconditionally added.
and if you find it more elegant, the same could be also done there depending on OMNECT_ETH[12...] being defined.

Reply:
i would prefer if OMNECT_ONLINE_INTERFACES is not semi-"dynamic" and i can see its value in the corresponding machine config.

Reply:
then there's no point in using OMNECT_ETH0 and friends any longer afaics, instead the devices can be directly written to OMNECT_ONLINE_INTERFACES in the machine config.

Reply:
changed